.github/workflows/e2e_translation_classroom_and_core_features.yml

```diff
@@ -30,6 +30,9 @@ jobs:
       - name: Install chrome
         if: startsWith(github.head_ref, 'update-changelog-for-release') == false
         run: python -m scripts.install_chrome_for_ci
+      - name: Install ffmpeg
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
```

Comment:
[Out of context of this PR]

@U8NWXD I'm not sure why we have job.steps.if in all the places, should we use [job.if](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idif)?

Reply:
This is going to be changed by my #12440 PR.

Reply:
In that case, we should define run_e2e_test as an action, maybe? (with `suite-name` as an arg for the action), that way we will have these if checks in just one file.

Reply:
I don't know how important it really is to keep all the if checks in one file since once #12440 merges, this check will be changed to checking a variable, so it won't have much logic. I do think we should have an action for installing all the dependencies, but I think that's for another PR.

Reply:
>  I do think we should have an action for installing all the dependencies

I was suggesting to have the action for running e2e test which will take suite as param and run the doo all perform all the required steps.

>, but I think that's for another PR.

Yep, this thread is out of context of this PR as mentioned in the first comment! :)